src/lib/util/validateKey.js

```diff
@@ -0,0 +1,6 @@
+export default function validateKey(object, key, errorMessage) {
+  if (object[key] === undefined) {
```

Comment:
```suggestion
 if(object[key]) {
      return true;
   }
   throw new Error(errorMessage);
```

Reply:
@Daniyal-Qureshi 
Thanks for your suggestion. But, **if(object[key])** will result **false** if object[key] is any **falsy** value like 0, "" (empty string), null, etc. This will defeat the purpose of the function **validateKey**.

Reply:
@KUSHAGRARAJTIWARI Previously, you were checking the value of the given key,I suggested the better way to check if the value is undefined, I think you need to check whether is key is present in the object or not.

Reply:
Yes, you are correct. I changed the code and am now using the hasOwnProperty() method. This method sends true if the key is present in the object irrespective of the corresponding value. Using Javascript utility functions is good for code.